lib/src/widgets/chat_groupedlist_widget.dart

```diff
@@ -93,15 +89,10 @@ class ChatGroupedListWidget extends StatefulWidget {
 }
 
 class _ChatGroupedListWidgetState extends State<ChatGroupedListWidget>
-    with SingleTickerProviderStateMixin {
+    with TickerProviderStateMixin {
```

Comment:
Here is a specific for using `TickerProviderMixin `because the value of animation is being used at child level so it considers multiple animations that is why we have added `TickerProviderMixin` instead `SingleTickerProviderStateMixin`